src/app/views/apps/modals/app-settings-modal.html

```diff
@@ -26,13 +26,23 @@
           </div>
         </div>
       </div>
-
       <!-- Add on settings -->
       <div ng-show="helper.isAddOnSettingShown(app)">
         <hr>
         <button ng-click="helper.addOnSettingLauch(app)" class="btn btn-warning">{{ 'mno_enterprise.templates.impac.dock.settings.access_add-on_settings' | translate }}</button>
       </div>
-
+      <!-- Billing Details -->
+      <div ng-show="helper.canDisplayBilling()">
+        <hr>
+        <div ng-show="!!app.free_trial_end_at">
+            {{ 'mno_enterprise.templates.impac.dock.settings.free_trial' | translate }}: {{ app.free_trial_end_at }}
+        </div>
+        <div ng-show="!app.free_trial_end_at && app.per_user_licence">
+          <ng-pluralize count="app.licences_count" when="{'0': '{{ &quot;mno_enterprise.templates.impac.dock.settings.no_users&quot; | translate }}', '1': '{{ &quot;mno_enterprise.templates.impac.dock.settings.1_user&quot; | translate }}', 'other': '{{app.licences_count | number}} {{ &quot;mno_enterprise.templates.impac.dock.settings.users&quot; | translate }}'}"
```

Comment:
I would suggest you add the MessageFormat plugin, using the bower command:
`bower install angular-translate-interpolation-messageformat`

Then add it to your mno-enterprise-angular/src/app/index.config.coffee file:
`$translateProvider.addInterpolation('$translateMessageFormatInterpolation');`
This means that it will use default ng-translate functionalities and only load this plugin when required.

You can then add this translation in the en.json file:
`mno_enterprise.templates.impac.dock.settings.nb_of_users: '{num_users, plural, =0{No users} =1{1 user} other{# users}}';`
1. 'num_users' - The name of the variable you are using to test the number
2. 'plural' - The type of action you want to use ( it could also be 'gender' for example)
3. All the different cases (# replace the variable value in your translation).

Then you can simply use it like this in your template
`<p 
translate="PLURAL" 
translate-values="{ num_users: app.licences_count }" 
translate-interpolation="messageformat"></p>`

If you wanna play around with it I made this plunkr:
http://plnkr.co/edit/GyrLvacl7rk6GgsH0S6o?p=preview

Let me know if you have any problem.
Here is the doc: https://angular-translate.github.io/docs/#/guide/14_pluralization